packages/embark-contracts-manager/src/index.js

```diff
@@ -107,21 +107,30 @@ class ContractsManager {
           contract: (callback) => {
             self.events.request('contracts:contract', req.body.contractName, (contract) => callback(null, contract));
           },
+          web3: (callback) => {
+            self.events.request("blockchain:get", web3 => callback(null, web3));
+          },
           account: (callback) => {
             self.events.request("blockchain:defaultAccount:get", (account) => callback(null, account));
           }
         }, (error, result) => {
           if (error) {
             return res.send({error: error.message});
           }
-          const {account, contract} = result;
+          const {account, contract, web3} = result;
           const abi = contract.abiDefinition.find(definition => definition.name === req.body.method);
           const funcCall = (abi.constant === true || abi.stateMutability === 'view' || abi.stateMutability === 'pure') ? 'call' : 'send';
 
           self.events.request("blockchain:contract:create", {abi: contract.abiDefinition, address: contract.deployedAddress}, async (contractObj) => {
             try {
-              const gas = await contractObj.methods[req.body.method].apply(this, req.body.inputs).estimateGas();
-              contractObj.methods[req.body.method].apply(this, req.body.inputs)[funcCall]({from: account, gasPrice: req.body.gasPrice, gas: Math.floor(gas)}, (error, result) => {
+              const value = req.body.value !== undefined ? web3.utils.toWei(req.body.value, 'ether') : 0
```

Comment:
Values sent through Cockpit are considered to be in `ether`. Embark converts it to Wei accordingly. We might want to make the unit configurable though.

Reply:
Just to be surte, sending a value of 0 works for non-payable functions too? Asking in case it requires `null`.
Also, ETH is fine for now (so long as we make it clear that it's ETH we require), but making it configurable would be even better (maybe just a dropdown with ETH vs Wei)